### DISCUSSION TITLE
Erro can't evaluate field LanguageCode in type *langs.Language

### DISCUSSION THREAD

**Discussion by willamecm83** [adityatelange/hugo-PaperMod#1437] [Q&A]
hello everyone, I'm using the hugo v0.123.3 version, I'm having problems, when I use the hugo server it gives the 404 page, or I try to build pages on github it gives an error (in the image)

![Captura de tela de 2024-02-24 23-27-01](https://github.com/adityatelange/hugo-PaperMod/assets/51059685/a0fd85e5-f435-4614-9148-5c7b38ceaa27)

in the file hugo.yaml

baseURL: "https://willamecm-10.github.io/hugofb/"
languageCode: en-uk
title: My New Hugo Site
theme: PaperMod

The error is practically this "can't evaluate field LanguageCode in type *langs.Language"

In the i18n folder there is uk I changed it to en-uk I still have the same error, I have no more ideas to solve it, can anyone help me

**Comment by adityatelange** [comment]
Hi did you miss the error on line 7?

**Comment by jasonbraganza** [comment]
> hello everyone, I'm using the hugo v0.123.3 version, 

@willamecm-10, line 6 in the log says, the runner is using `hugo v0.110.0`. Maybe you need tell them what version to use via some setting / environment variable

**Reply by willamecm83** [reply]
Now I'm using this script 

https://gohugo.io/hosting-and-deployment/hosting-on-github/

but it's stopping at a protection rule

![Captura de tela de 2024-02-25 10-37-12](https://github.com/adityatelange/hugo-PaperMod/assets/51059685/5f54c111-2f0e-425a-b8da-b6450630560b)

Is First time I'm trying to deploy, I was following a step by step guide on YouTube

I got the old script from YouTube, I was following it step by step.

As it wasn't working, I found this one from Hugo himself

**Reply by willamecm83** [reply]
I found the answer about deploy rules

https://github.com/orgs/community/discussions/39054#discussioncomment-6420042 

now working correctly

![Captura de tela de 2024-02-25 11-01-47](https://github.com/adityatelange/hugo-PaperMod/assets/51059685/09650076-99b2-4a5d-ae9a-93ed2d040073)

**Comment by willamecm83** [comment]
the error was the deploy script.

But now it works, thanks for responding

@jasonbraganza @adityatelange
